focus-autosave-mode.el

```diff
@@ -73,12 +75,12 @@
 (defun focus-autosave-save-marked ()
   "Save the marked buffers and remove the killed ones from the list."
   (setq focus-autosave-buffer-list
-        (delete-if-not #'buffer-live-p
-                       focus-autosave-buffer-list))
-  (mapcar (lambda (buffer)
-            (with-current-buffer buffer
-              (save-buffer)))
-          focus-autosave-buffer-list)
+        (cl-delete-if-not 'buffer-live-p
```

Comment:
Use `cl-delete-if-not` in favour of `delete-if-not` to avoid warnings about `cl` being used at runtime.